common/src/java/org/apache/hadoop/hive/common/type/Date.java

```diff
@@ -39,7 +39,7 @@
 /**
  * This is the internal type for Date. The full qualified input format of Date
  * is "uuuu-MM-dd". For example: "2021-02-11".
- * <table border="2">
+ * <table border="2" summary="">
```

Comment:
jeez! this problem will make a comeback in jdk11 :facepalm: 
https://stackoverflow.com/questions/22528767/how-to-work-around-the-stricter-java-8-javadoc-when-using-maven